test/junit/scala/lang/stringinterpol/StringContextTest.scala

```diff
@@ -72,7 +72,7 @@ class StringContextTest {
 
   // verifying that the standard interpolators can be supplanted
   @Test def antiHijack_?() = {
-    object AllYourStringsAreBelongToMe { case class StringContext(args: Any*) { def s(args: Any) = "!!!!" } }
+    object AllYourStringsAreBelongToMe { case class StringContext(args: Any*) { def s(args: Any*) = "!!!!" } }
```

Comment:
The other thing I always accidentally do, is forward to `s` with `s(args)` instead of `s(args: _*)`.